e2e/__snapshots__/definition.test.ts.snap

```diff
@@ -12,11 +12,11 @@ exports[`Definition Definition on parameter 2`] = `
     "range": {
       "end": {
         "character": 28,
-        "line": 465,
+        "line": 481,
       },
       "start": {
         "character": 12,
-        "line": 462,
+        "line": 478,
```

Comment:
Why did these need updating?

Reply:
That's a very good question, it seems as the tested file uses the `mac` orb, the E2E snapshots also records the orb's diagnostics. It seems the orb has been updated and so the E2E tests also need to be updated.

What's really strange is why does the E2E tests check the `mac` orb? I haven't taken time to look into but we definitely should